### PR TITLE
Add App Center CI/CD Integration Instruction UI in Marketplace

### PR DESCRIPTION
**Pull request by Parthipan-Natkunam**
Add "App Center" UI card and instruction modal template.

JIRA ID: [AP-440](https://appknox.atlassian.net/browse/AP-440)

Change Log:
---
1. Add a new card for "App Center" integration in the marketplace section.
2. Add a new template literal for the instruction model pop-up content.

### CODE REVIEW

File: app/components/marketplace-plugin-list/constants.js [hunk 1]
```diff
@@ -35,5 +35,40 @@ $ appknox upload &lt;path to apk/ipa file&gt; | xargs appknox cicheck --risk-thr
...
     </ul>
-  `
+  `,
+appCenterInstructions:`
+  <ul class="bullet-list margin-r-1 margin-l-1h">
+    <li class="margin-b-1">
+      Generate a personal access token from <a href="/settings/developersettings" target="_blank">Developer Settings</a>
+    </li>
+    <li class="margin-b-1">
+      Create a file <code class="black-text">appcenter-post-build.sh</code> in your source code repository and add the content given below:
```
Comment (sks444) [inline app/components/marketplace-plugin-list/constants.js]: Should we specify in what directory the file needs to be created? E.g. for Android its: `/app/appcenter-post-build.sh` and for IOS it's `/appcenter-post-build.sh`.

File: app/components/marketplace-plugin-list/constants.js [hunk 2]
```diff
@@ -35,5 +35,40 @@ $ appknox upload &lt;path to apk/ipa file&gt; | xargs appknox cicheck --risk-thr
...
+        curl -L https://github.com/appknox/appknox-go/releases/download/1.1.0/appknox-\`uname -s\`-x86_64 > appknox && chmod +x appknox
+        ./appknox upload $APPCENTER_OUTPUT_DIRECTORY/app-release.apk
+        rm appknox
+  else
+      echo "Current branch is $APPCENTER_BRANCH"
+  fi
+fi</pre>
+      <small>(More info: <a href="https://aka.ms/docs/build/custom/scripts" target="_blank">https://aka.ms/docs/build/custom/scripts)</a></small>
+      <p class="margin-t-1 alert-info padding-10">
+          Make sure you use the latest version of the appknox binary in the above curl URL
```
Comment (sks444) [inline app/components/marketplace-plugin-list/constants.js]: We are not mentioning this warning in other instructions so we can skip it here also.

As we will specifically be informing our customers whenever major releases happens.

File: app/components/marketplace-plugin-list/constants.js (hunk 1 above)
Comment (Parthipan-Natkunam) [reply app/components/marketplace-plugin-list/constants.js]: Yes. that will give more clarity. Have done the change.

File: app/components/marketplace-plugin-list/constants.js (hunk 2 above)
Comment (Parthipan-Natkunam) [reply app/components/marketplace-plugin-list/constants.js]: Done. Removed it.

**Pull request merged**